apps/ledger-live-desktop/src/renderer/bridge/SyncNewAccounts.ts

```diff
@@ -14,6 +14,9 @@ export const SyncNewAccounts = ({ priority }: { priority: number }) => {
         type: "SYNC_SOME_ACCOUNTS",
         accountIds,
         priority,
+        // TODO: reason was missing here - but the types require it
+        // so I'm adding it as a falsy empty string
+        reason: "",
```

Comment:
please use `"sync-new-account"` because that field is used to identify "why" a sync happens for stats